app/src/main/AndroidManifest.xml

```diff
@@ -9,12 +9,15 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+
+         <activity android:name="ch.epfl.polybazaar.FillListingActivity"></activity>
```

Comment:
Here we should add a parent activity (maybe the main activity for now?) where we can select to go to the filling screen or the overview screen